cmd/containers-storage/diff.go

```diff
@@ -171,7 +171,7 @@ func applyDiffUsingStagingDirectory(flags *mflag.FlagSet, action string, m stora
 		file: tar,
 	}
 
-	differ, err := chunked.GetDiffer(context.Background(), m, size, metadata, &fetcher)
+	differ, err := chunked.GetDiffer(context.Background(), m, nil, size, metadata, &fetcher)
```

Comment:
Non-blocking? This debug-only command could take the trouble to compute the digest, and then the generic code would not have that risky extra condition for “digest is not present”.